core/domain/email_manager.py

```diff
@@ -101,27 +101,35 @@ def log_new_error(*args, **kwargs):
 
 EXPLORATION_ROLE_MANAGER = 'manager rights'
 EXPLORATION_ROLE_EDITOR = 'editor rights'
+EXPLORATION_ROLE_TRANSLATOR = 'translator rights'
 EXPLORATION_ROLE_PLAYTESTER = 'playtest access'
 
 EDITOR_ROLE_EMAIL_HTML_ROLES = {
     rights_manager.ROLE_OWNER: EXPLORATION_ROLE_MANAGER,
     rights_manager.ROLE_EDITOR: EXPLORATION_ROLE_EDITOR,
+    rights_manager.ROLE_TRANSLATOR: EXPLORATION_ROLE_TRANSLATOR,
     rights_manager.ROLE_VIEWER: EXPLORATION_ROLE_PLAYTESTER
 }
 
 _EDITOR_ROLE_EMAIL_HTML_RIGHTS = {
     'can_manage': '<li>Change the exploration permissions</li><br>',
     'can_edit': '<li>Edit the exploration</li><br>',
+    'can_translate': '<li>Translate the exploration</li><br>',
     'can_play': '<li>View and playtest the exploration</li><br>'
 }
 
 EDITOR_ROLE_EMAIL_RIGHTS_FOR_ROLE = {
```

Comment:
Add comment above here: "We don't include "can_translate" for managers and editors, since this is implied by edit rights".

Reply:
Done!